From 9bacf5a1bf079c972abd66b3bd8bca923ae6ecb1 Mon Sep 17 00:00:00 2001
From: Sebastian Keller <sebastian-keller@gmx.de>
Date: Sat, 27 Jun 2020 17:21:53 +0200
Subject: [PATCH] tools: Fix joystick property matching in udev rule

This was not doing actual matching of the property, but assigning it,
which resulted in no device being assigned this property. This broke
gamepad/joystick permissions.

Fixes #271
---
 tools/65-libwacom.rules.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/65-libwacom.rules.in b/tools/65-libwacom.rules.in
index e7ce4aa..04f2bf3 100644
--- a/tools/65-libwacom.rules.in
+++ b/tools/65-libwacom.rules.in
@@ -20,6 +20,6 @@ KERNELS=="input*", \
 
 # We can't unset properties through the hwdb but we can set them to zero.
 # So let's have a rule that converts the 0 properties to unset ones.
-ENV{ID_INPUT_JOYSTICK}="0", ENV{ID_INPUT_JOYSTICK}=""
+ENV{ID_INPUT_JOYSTICK}=="0", ENV{ID_INPUT_JOYSTICK}=""
 
 LABEL="libwacom_end"
